src_assets/common/assets/web/PlatformLayout.vue

```diff
@@ -0,0 +1,27 @@
+<script setup>
+const props = defineProps({
+  platform: {
+    type: String,
+    required: true
+  }
+})
+
+</script>
+<template>
+<template v-if="$slots.windows && platform === 'windows'">
+  <slot name="windows"></slot>
+</template>
+
+<template v-if="$slots.linux && platform === 'linux'">
+  <slot name="linux"></slot>
+</template>
+
+<template v-if="$slots.macos && platform === 'macos'">
+  <slot name="macos"></slot>
+</template>
+</template>
```

Comment:
```suggestion
<template>
  <template v-if="$slots.windows && platform === 'windows'">
    <slot name="windows"></slot>
  </template>

  <template v-if="$slots.linux && platform === 'linux'">
    <slot name="linux"></slot>
  </template>

  <template v-if="$slots.macos && platform === 'macos'">
    <slot name="macos"></slot>
  </template>
</template>
```

I'm not a vue expert, but I think it would be more readable if things were nested.

Reply:
Sorry for the mess with formatting, using the same IDE for C++, Vue, etc, editor rules where a mess opening everything as a single project. I'm not an expert too, but I fully agree.

Reply:
Hmm... You might consider CLion (although paid), it can handle different file formats quite easily within the same project.

They do offer a free license for open source developers though. https://www.jetbrains.com/community/opensource/